ng/src/web/pages/agents/detailspage.js

```diff
@@ -101,6 +102,12 @@ const ToolBarIcons = withCapabilities(({
           onClick={onAgentVerifyClick}
         />
       }
+      <Icon
+        img="agent.svg"
+        value={entity}
+        title={_('Download Agent installer package')}
```

Comment:
I'd go for lower case "agent", because for the user it's really just a noun, not the component.